docs/developers/protocol/hermez-protocol/protocol.md

```diff
@@ -95,6 +95,7 @@ dataBuffer: [48 bits] fromIdx
 - `MAX_TX`: absolute maximum L1 or L2 transactions allowed to process in one batch
 - `MAX_L1_TXS`: absolute maximum of L1 transactions allowed to process in one batch
 - `CHAIN_ID`: identifier zkRollup chain
+- `CURRENT_BATCH_NUMBER`: current batch number processed
```

Comment:
Why is this variable in uppercase?  I understand that:
1. It's not a constant
2. It's not a parameter of the circuit (but an input).

Also, why is it under Global Settings?

Reply:
it is an input forced by the smart contract, so I think it fits on global settings but in `contract` section. (same applies for `chain_id` which should be also in lowercase).